.github/workflows/release.yml

```
@@ -244,10 +244,13 @@ jobs:
             'select(.kind == "Deployment") | .spec.template.metadata.annotations["io.katacontainers.config.agent.policy"]' |
             base64 -d | sha256sum | cut -d " " -f1 > cli/cmd/assets/coordinator-policy-hash
 
+          cp image-replacements.txt cli/cmd/assets/image-replacements.txt
+
           git config --global user.name "edgelessci"
           git config --global user.email "edgelessci@users.noreply.github.com"
           git add cli/cmd/assets/coordinator-policy-hash
-          git diff --staged --quiet || git commit -m "release: update coordinator policy hash"
+          git add cli/cmd/assets/image-replacements.txt
+          git diff --staged --quiet || git commit -m "release: update coordinator policy hash and embed image replacements"
```

```suggestion
          git diff --staged --quiet || git commit -m "release: update cli assets"
```
... in anticipation of other future assets.